app/src/main/java/nz/org/cacophony/birdmonitor/Util.java

```diff
@@ -815,7 +854,7 @@ public static void createFailSafeAlarm(Context context) { // Each alarm creates
 
     public static long getMilliSecondsBetweenRecordings(Prefs prefs) {
         if (prefs.getUseVeryFrequentRecordings()) {
-            return (long) prefs.getTimeBetweenVeryFrequentRecordingsSeconds();
+            return (long) prefs.getTimeBetweenVeryFrequentRecordingsSeconds() * 1000;
```

Comment:
Why this change? The method still says "seconds"

Reply:
It is retrieving seconds but we want milliseconds.